objax/module.py

```diff
@@ -53,6 +54,66 @@ def __call__(self, *args, **kwargs):
         raise NotImplementedError
 
 
+class ForceArgs(Module):
+    """Forces override of arguments of given module."""
+
+    ANY = namedtuple('Any', ())
+    """Token used in `ForceArgs.undo` to indicate undo of all values of specific argument."""
+
+    @staticmethod
+    def _remove_args(force_args: 'ForceArgs', args_to_remove: Dict[str, Any]):
```

Comment:
I just realized this should probably not be a static method.
```suggestion
    def remove_args(self, args_to_remove: Dict[str, Any]):
```